src/index.js

```diff
@@ -82,7 +82,7 @@ class UglifyJsPlugin {
 						let ast = uglify.parse(input, {
 							filename: file
 						});
-						if(options.compress !== false) {
+						if(options.compress !== false && options.compress !== undefined) {
```

Comment:
`if (options.compress) {}` ? 

Is there a reason for checking those separately? A edge case ?

Reply:
What I understand it shouldn't be called 3 times - only 1 time - so this case should be investigated. I suppose the real problem in this multiple calls and not in check that I proposed.